engine.py

```diff
@@ -912,3 +912,58 @@ def __setup_favorite_dirs(self):
                                 type=(nuke.IMAGE | nuke.SCRIPT | nuke.GEO),
                                 icon=icon_path,
                                 tooltip=path)
+
+    #####################################################################################
+    # Script and session related methods
+
+    def get_session_path(self):
```

Comment:
This does not reflect the core interface - a `session=None` parameter is missing!